plugins/py3embed/py3embed.cpp

```diff
@@ -381,11 +485,11 @@ static class Python3xGlobalState
         // and if we add our own lock we are liable to deadlock as the code within Py_CompileStringFlags may
         // temporarily release then re-acquire the GIL.
         if (!activityContextTupleType)
-            activityContextTupleType.setown(getNamedTupleType("isLocal,numSlaves,numStrands,slave,strand"));
+            activityContextTupleType.setown(getNamedTupleType("isLocal,numSlaves,numStrands,slave,strand", 5));
         return activityContextTupleType.get();
     }
 
-    PyObject *getNamedTupleType(const char *names)
+    PyObject *getNamedTupleType(const char *names, unsigned numFields)
```

Comment:
numFields not actually used....

Reply:
No, that was part of my first failed attempt to fix.I will remove.